packages/cashier/src/types/websocket.types.ts

```diff
@@ -52,6 +52,15 @@ type TServiceTokenResponse = {
     };
 };
 
+export interface TradingPlatformStatus {
```

Comment:
We're using `type` in our code base. We'd better not use `interface` since it was agreed that we use `type`.